edition/rd

```diff
@@ -6,4 +6,6 @@ LIMA_INSTALL_IPTABLES=true
 LIMA_INSTALL_LIMA_INIT=true
 LIMA_INSTALL_LOGROTATE=true
 LIMA_INSTALL_NERDCTL=true
+LIMA_INSTALL_CONTAINERD=false
```

Comment:
Rancher Desktop uses containerd as compiled into k3s, rather than the regular containerd package.

Reply:
Note that `LIMA_INSTALL_DOCKER` implies `LIMA_INSTALL_CONTAINERD`. So the RD edition has containerd installed, even if it may not use it.

Reply:
"It's complicated". There are no guarantees that the containerd (or runc) used by docker is available to others... It might be on a special path, and it might be an older version. So I would prefer if no such implication would be made between them.